app/controllers/dashboards_controller.rb

```diff
@@ -42,14 +42,19 @@ def push
     if version.blank?
       flash.alert = 'No preview version available'
     else
-      pv = Version.create(number: version.number, production: true, user: current_user)
+      production_version = Version.create(number: version.number, production: true, user: current_user)
 
-      if pv.persisted?
+      if production_version.persisted?
         flash.notice = 'Production data updated'
 
         # Build Sitemap and notify search engines in production only
         ping = request.original_url.include?(GibctSiteMapper::PRODUCTION_HOST)
         GibctSiteMapper.new(ping: ping)
+
+        if Settings.institutions.archive
+          # Archive old institution rows
+          InstitutionsArchive.archive_previous_versions()
```

Comment:
nitpick, but no need for trailing `()`

Reply:
yeah, rubocop should [catch this](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/MethodCallWithoutArgsParentheses)

Reply:
fixed